examples/scenes/src/test_scenes.rs

```diff
@@ -1810,6 +1812,16 @@ mod impls {
     pub(super) fn image_extend_modes(scene: &mut Scene, params: &mut SceneParams) {
         params.resolution = Some(Vec2::new(1500., 1500.));
         params.base_color = Some(palette::css::WHITE);
+        image_extend_modes_helper(scene, ImageQuality::Medium);
+    }
+
+    pub(super) fn image_extend_modes_nearest_neighbor(scene: &mut Scene, params: &mut SceneParams) {
```

Comment:
We could impl this in a way similar to `longpathdash`, which is another parameterised test scene